tests/resources/test-types.json

```diff
@@ -1343,6 +1345,7 @@
   {
     "id": "94",
     "linkedIds": null,
+    "suggestedTestTypeIds": ["53", "54", "94"],
```

Comment:
I would suggest to add unit test for those if possible so we cover our schema.
We could add an unit that calls the endpoint and then assert that the object contains
```json
    "id": "40",
    "linkedIds": null,
    "suggestedTestTypeIds": ["40", "53", "54", "199"],
    "name": "Annual test",
    "testTypeName": "Annual test",
    "forVehicleType": ["trl"],
    ...
```

it will warn us in case we get to change the schema and also verify our API specs.

We could be using the `toMatch` or toMatchObject` from jest I believe.

Reply:
I have now created this test